example/src/Modelling/PetriNet/PetriReach/Config.hs

```diff
@@ -84,5 +87,69 @@ task2024_60 = ReachConfig {
   showLengthHint = False,
   showMinLengthHint = True,
   showTargetNet = True,
-  showPlaceNamesInNet = False
+  showPlaceNamesInNet = False,
+  filterConfig = defaultFilterConfig
+  }
+
+{-|
+Example configuration with custom filter settings to exclude trivial sequences.
+This configuration is more strict about filtering patterns to prevent students
+from accidentally guessing correct answers.
+-}
+taskWithStrictFiltering :: ReachConfig
+taskWithStrictFiltering = ReachConfig {
+  netGoalConfig = NetGoalConfig {
+    numPlaces = 4,
+    numTransitions = 4,
+    capacity = Unbounded,
+    drawCommands = [Circo],
+    maxTransitionLength = 8,
+    minTransitionLength = 8,
+    postconditionsRange = (2, Just 3),
+    preconditionsRange = (2, Just 2)
+    },
+  printSolution = True,
+  rejectLongerThan = Nothing,
+  showLengthHint = True,
+  showMinLengthHint = True,
+  showTargetNet = True,
+  showPlaceNamesInNet = True,
+  filterConfig = FilterConfig {
+    filterCyclicPatterns = True,         -- Reject [t1,t2,t3,t4,t1,t2,t3,t4]
+    filterRepetitiveSubsequences = True, -- Reject [t4,t4,t4,t4,...]
+    filterGroupedRepeats = True,         -- Reject [t1,t1,t2,t2,t3,t3,t4,t4]
+    minRepetitiveLength = 2,             -- More strict: 2+ consecutive repeats
+    maxCycleLength = 3                   -- Check smaller cycles only
+    }
+  }
+
+{-|
+Example configuration with filtering disabled for comparison purposes.
+Use this when you want the traditional behavior without filtering.
+-}
+taskWithoutFiltering :: ReachConfig
+taskWithoutFiltering = ReachConfig {
+  netGoalConfig = NetGoalConfig {
+    numPlaces = 4,
+    numTransitions = 4,
+    capacity = Unbounded,
+    drawCommands = [Circo],
+    maxTransitionLength = 8,
+    minTransitionLength = 8,
+    postconditionsRange = (2, Just 3),
+    preconditionsRange = (2, Just 2)
+    },
+  printSolution = True,
+  rejectLongerThan = Nothing,
+  showLengthHint = True,
+  showMinLengthHint = True,
+  showTargetNet = True,
+  showPlaceNamesInNet = True,
+  filterConfig = FilterConfig {
+    filterCyclicPatterns = False,
+    filterRepetitiveSubsequences = False,
+    filterGroupedRepeats = False,
+    minRepetitiveLength = 3,
+    maxCycleLength = 4
+    }
```

Comment:
I guess these two configs should be dropped from the file? They are not used in any way for testing or so.